src/ray/gcs/redis_gcs_client.cc

```diff
@@ -114,6 +123,8 @@ Status RedisGcsClient::Connect(boost::asio::io_service &io_service) {
                                              /*password=*/options_.password_));
   }
 
+  Attach(io_service);
```

Comment:
Should we still check the status here?

Reply:
Because the return status of `Attach` is always ok, so I change the return status to void.